pages/index.tsx

```diff
@@ -1,83 +1,88 @@
-import { ConnectButton } from '@rainbow-me/rainbowkit';
-import type { NextPage } from 'next';
-import Head from 'next/head';
-import styles from '../styles/Home.module.css';
+import { Button } from "@chakra-ui/button";
+import { Input, InputGroup, InputRightElement } from "@chakra-ui/input";
+import { Box, Flex, Heading, Text } from "@chakra-ui/layout";
+import { Select } from "@chakra-ui/select";
+import {
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  FormHelperText,
+} from "@chakra-ui/form-control";
+import type { NextPage } from "next";
+import Head from "next/head";
+
+const exampleTokenList = [
```

Comment:
is there any sense in trying to componentize more of this index page? reluctant to over-engineer given that it's an SPA, curious for your thoughts @alexkeating

Reply:
Yes, definitely think we should componentize more. My goal with this pr was just to put in a scaffold, and I figured we could handle pulling out these components as we added interactive functionality as any extra work now would probably be redone.